Source/ViewControllers/BrickCollectionView.swift

```diff
@@ -234,7 +234,27 @@ open class BrickCollectionView: UICollectionView {
             self.collectionViewLayout.invalidateLayout(with: BrickLayoutInvalidationContext(type: .invalidate))
         }, completion: completion)
     }
-
+    
+    open func resizeBrick(with identifier: String, newSize: BrickSize, completion: ((Bool) -> Void)? = nil)  {
+        self.performBatchUpdates({
+            if let indexPath = self.indexPathsForBricksWithIdentifier(identifier).first {
```

Comment:
The index path may have changed after the invalidateLayout so we have to get it again